dashboard/app/helpers/levels_helper.rb

```diff
@@ -476,8 +476,7 @@ def firebase_options
 
   def azure_speech_service_options
     speech_service_options = {}
-
-    if @level.game.use_azure_speech_service? && !CDO.azure_speech_service_region.nil? && !CDO.azure_speech_service_key.nil?
+    if @level.game.use_azure_speech_service? && CDO.azure_speech_service_region.present? && CDO.azure_speech_service_key.present?
```

Comment:
Did you intend to no longer protect against `nil`? I'm not a Ruby expert but I just tried `nil.present?` and got `NoMethodError (undefined method `present?' for nil:NilClass)`

Reply:
you're totally right, I just switched it back